package.json

```diff
@@ -30,7 +30,8 @@
     "test-ts": "rm -rf */dist && tsc --build web-ts native"
   },
   "resolutions": {
-    "moment": "2.29.1"
+    "moment": "2.29.1",
+    "react": "16.13.1"
```

Comment:
Why is this necessary? Would really love it to be removed, if possible.

Reply:
It is necessary to define hooks in the api-client.